eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -785,14 +793,13 @@ object Router {
   /**
     * https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md#clarifications
     */
-  val ROUTE_MAX_LENGTH = 20
+  val DEFAULT_ROUTE_MAX_LENGTH = 20
 
   // The default amount of routes we'll search for when findRoute is called
   val DEFAULT_ROUTES_COUNT = 3
 
-  // The default allowed 'spread' between the cheapest route found an the others
-  // routes exceeding this difference won't be considered as a valid result
-  val DEFAULT_ALLOWED_SPREAD = 0.1D
+  // The default params used for route searching, can be overridden by the caller
+  val DEFAULT_ROUTE_PARAMS = RouteParams(maxFeeBaseMsat = 21000, maxFeePct = 0.03, routeMaxCltv = 144, routeMaxLength = 6)
```

Comment:
These parameters should be in our configuration file.

Reply:
@sstone done in 4f7efc3. I took the chance to group the router configurations into one case class for convenience, the configuration values will be used as default unless overridden by call specific parameters. Also I came up with a 1 week (1008 blocks) max CLTV for routes although the spec doesn't define a max CLTV for that.